after/syntax/css/vim-coloresque.vim

```diff
@@ -122,9 +122,11 @@ function! s:VimCssInit(update)
     if a:update==1
         call s:ClearMatches()
     endif
-    :set isk+=-
-    :set isk+=#
-    :set isk+=.
+    if 'css' == &filetype
```

Comment:
This is a different bug with 3 other open pull requests fixing it. Revert this back so you don't get merge conflicts.